### PR TITLE
Resource naming and variables

### PR DESCRIPTION
**Pull request by venkatsaik**
Resource naming and variables.

### CODE REVIEW

File: terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,4 @@
+env_code     = "terraform-variables"
+vpc_cidr     = "10.0.0.0/16"
+public_cidr  = ["10.0.0.0/24", "10.0.1.0/24"]
+private_cidr = ["10.0.2.0/24", "10.0.3.0/24"]
```
Comment (geoffrey1330) [inline terraform.tfvars]: Add newline at the end of file

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: terraform.tfvars (hunk 1 above)
Comment (venkatsaik) [reply terraform.tfvars]: Completed.

**Review by geoffrey1330 — APPROVED** [review]

**Comment by venkatsaik** [comment]
Done.

**Comment by venkatsaik** [comment]
Merged to Main branch.

**Pull request merged**